roles/openshift_aws/defaults/main.yml

```diff
@@ -46,6 +46,21 @@ openshift_aws_ami_tags:
 openshift_aws_s3_mode: create
 openshift_aws_s3_bucket_name: "{{ openshift_aws_clusterid }}-docker-registry"
 
+openshift_aws_vpc_tags:
+  Name: "{{ openshift_aws_vpc_name }}"
+
+openshift_aws_vpc:
+  name: "{{ openshift_aws_vpc_name }}"
+  cidr: 172.31.0.0/16
+  subnets:
+    us-east-1:
+    - cidr: 172.31.48.0/20
+      az: "us-east-1c"
+#    - cidr: 172.31.32.0/20
```

Comment:
Should these additional azs not be re-introduced now that we're considering it to be an ordered list?

Reply:
They need to be commented until multi-az is enabled since subnets/elbs/asg's now loop through this dict.  

Here is the expected behavior....
- Existing clusters / single-az: Three subnets would be deployed previously but only default_az (now element 0) would be used by ASG's and ELB's.  The other 2 subnets would be deployed across az' and continue to hand around unused.  We don't want a change to that behavior.
- New clusters / single-az: One subnet would be deployed per element 0 and be used by ASG's and ELB's.
- New multi-az clusters: Uncomment additional dictionary elements then run plays.  Three subnets would be deployed spanning AZ's but fail if more az's are requested than avail in the region.  Subnets will be utilized by ASG's and ELB's.
- Upgrade single-az to multi-az: Uncomment additional dictionary elements then run plays.   Subnets get ensured.   There won't be any changes if subnets under a count of 3 are requested as they were previously landed.  If 3+ are requested then subnets would be deployed spanning the AZ's but fail if more az's are requested than avail in the region.  ASG's and ELB's will be updated to utilize additional subnets.

Reply:
Ok, thanks for the explanation.